amazonorders/lib/transactions.py

```diff
@@ -0,0 +1,62 @@
+__copyright__ = "Copyright (c) 2024 Jeff Sawatzky"
+__license__ = "MIT"
+
+import datetime
+from typing import Dict, List, Optional, Tuple
+
+from bs4 import Tag
+
+from amazonorders.conf import AmazonOrdersConfig
+from amazonorders.entity.transaction import Transaction
+
+
+def parse_transaction_form_tag(
+    form_tag: Tag, config: AmazonOrdersConfig
+) -> Tuple[List[Transaction], Optional[str], Optional[Dict[str, str]]]:
+    transactions = []
+    date_container_tags = form_tag.select(
+        config.selectors.TRANSACTION_DATE_CONTAINERS_SELECTOR
+    )
+    for date_container_tag in date_container_tags:
+        date_tag = date_container_tag.select_one(
+            config.selectors.FIELD_TRANSACTION_COMPLETED_DATE_SELECTOR
+        )
+        assert date_tag is not None
```

Comment:
Instead of an assert here, let's log a warning and `continue`, since this Transaction can't be parsed.